### DISCUSSION TITLE
Home Assistant Side bar link to remote Frigate

### DISCUSSION THREAD

**Discussion by shanekuz** [blakeblackshear/frigate#11086] [Ask A Question]
### What is your question:

I would like to move my frigate install to a Pi4 with with Coral USB away from my home assistant install.  I really like the side bar in home assistant showing the frigate panel though and this only comes with the integration on HA.  Has anyone setup a frigate side bar pointing to a remove frigate?

**Comment by NickM-27** [comment]
Yes, this is accomplished with the frigate proxy addon

**Reply by shanekuz** [reply]
Thanks Nick, Works perfect.
